src/types.rs

```diff
@@ -1480,3 +1480,11 @@ impl SubscriptionFilter {
         self
     }
 }
+
+/// A projetion mode.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum ProjectionMode {
```

Comment:
**CONCERN:** I think that based on the RFC, this enum is redundant.

Reply:
Done